packages/x-components/src/x-modules/history-queries/store/__tests__/actions.spec.ts

```diff
@@ -355,7 +355,7 @@ describe('testing history queries module actions', () => {
       expectHistoryQueriesToEqual([{ ...gato, totalResults }, perro]);
     });
 
-    it('does not update a history query if the new totalResults is lower', async () => {
+    it('updates a history query if the new totalResults is lower', async () => {
```

Comment:
Hi! I'm not sure if this test definition matches the condition that is being tested. Should not it be related to this one: `* 3. If there is a new search response.`?

Reply:
Hi @annacv!! This test definition matches with the condition that is being tested because the number of totalResults of the response is 2 and totalResults saved into history query ("gato") are set to 50. 
So `"new totalResults"(2) < "historyQuery totalResults" (50)`

![Screenshot 2023-10-11 at 08 14 54](https://github.com/empathyco/x/assets/114984466/aa00b80c-7ffb-4504-aef0-4ac657a33847)

Related to your question, I think it's not necessary to make a test with this condition `3. If there is a new search response`, because when we test if the new totalResults are lower or higher than historyQuery totalResults, we are testing the totalResults of a new search response.

Reply:
Hi, @lauramargar, I meant adding a test to show that History Queries are being saved with filters 👀 .
And, about the definition, I was willing if it should be something like "updates a history query with a new response", I'll ping you to check if I'm lost in some way.